<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/02_image/background.jpg);</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/02_image/input.png);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_8">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_9">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/number.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_11">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/number.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_1">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_12">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/number.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_0">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_15">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/operator.ico);
color: rgb(255, 255, 0);
border-image: url(:/02_image/number.ico);</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_14">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/number.ico);
color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_13">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/02_image/number.ico);</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>lineEdit</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_15</zorder>
  <zorder>pushButton_4</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
